spec/models/comment_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:user) { create(:user) }
+  let(:post) { create(:post) }
+  let(:comment) { create(:comment, user:, post:) }
+
+  it 'is valid with valid attributes' do
+    expect(comment).to be_valid
+  end
+
+  it 'belongs to a user' do
+    expect(comment.user).to eq(user)
+  end
+
+  it 'belongs to a post' do
+    expect(comment.post).to eq(post)
+  end
+
+  it 'increments the comments_counter of the associated post after save' do
+    expect { comment.save }.to change { post.reload.comments_counter }.by(1)
+  end
+end
```

Comment:
- [ ] These tests are failing as well as the other files. Please see the screenshot below for reference. I encourage you to work on the tests to pass before submitting for review 👍🏽

<img width="1183" alt="Screenshot 2023-10-27 at 12 16 26 AM" src="https://github.com/Wineshuga/pen-it/assets/50941074/3b747f0f-033f-4e0c-8db9-64f172b4f5c2">